samples/new-node.yaml

```diff
@@ -0,0 +1,17 @@
+# Use this manifest to announce the new node to the apiserver.
+# You need to fill the needed SSH connection information.
+# CMD: sudo kubectl --kubeconfig /etc/kubernetes/admin.conf create -n "kube-system" -f new-node.yaml
+apiVersion: v1
+kind: Node
+metadata:
+  name: <NODE_NAME>
+  annotations:
+    metalk8s.scality.com/ssh-user: <SSH_USER>
+    metalk8s.scality.com/ssh-port: <SSH_PORT>
+    metalk8s.scality.com/ssh-host: <IP_ADDRESS>
```

Comment:
Would be useful to document here what the defaults are, and explain not all fields are, as such, mandatory.